Source/Components/RModel.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  RModel.swift
+//  RiveRuntime
+//
+//  Created by Zachary Duncan on 3/24/22.
+//  Copyright © 2022 Rive. All rights reserved.
+//
+
+import Foundation
+
+public struct RModel { // TODO: Rename to RAssetState
+    /// The web address from which to load the `RiveFile`
+    public let webURL: String? = nil
```

Comment:
nit, but why not just make this `url`?

Reply:
I considered calling it just `url` but since the `URL` type in Foundation can refer to local or remote locations I thought`webURL` would make it more clear that this is being used to find a remote location